### PR TITLE
Remove deprecated Stream APIs

### PR DESCRIPTION
**Pull request by pixelflinger**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ A new header is inserted each time a *tag* is created.
 ## v1.21.1 (currently main branch)
 
 - engine: allow both screen-space refraction and screen-space reflections on the same object [⚠️ **Material breakage**].
+- remove deprecated `Stream` APIs [⚠️ **API Change**].
```
Comment (bejado) [inline RELEASE_NOTES.md:9]: Please rebase on main. This should go under v1.21.2.

**Review by bejado — CHANGES_REQUESTED** [review]

**Comment by kpeeters** [comment]
This means that you now require API 26, right? Any way you might reconsider this decision? It removes quite a large chunk of devices still out there.

**Review by prideout — COMMENTED** [review]
We need to update `StreamHelper.kt` and make sure that `sample-stream-test` still builds and runs.

**Comment by pixelflinger** [comment]
> We need to update `StreamHelper.kt` and make sure that `sample-stream-test` still builds and runs.

How come everything built?

**Comment by pixelflinger** [comment]
> This means that you now require API 26, right? Any way you might reconsider this decision? It removes quite a large chunk of devices still out there.

Sorry why is that? I think the solution that uses `import()` works everywhere.

**Review by bejado — APPROVED** [review]

**Review by prideout — APPROVED** [review]

**Comment by pixelflinger** [comment]
converted to draft until we're ready to submit.

**Comment by bejado** [comment]
Please rebase on main and update the release notes for 1.21.3

**Pull request merged**